Datadog.Sketches.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32210.308
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Datadog.Sketches", "src\Sketches\Datadog.Sketches\Datadog.Sketches.csproj", "{5F7E8132-00F1-4DD7-868F-3522ECD28E05}"
```

Comment:
Do we want that `Sketches` sub folder? 🤔 Shouldn't it just be `src/Datadog.Sketches/Datadog.Sketches.csproj`?

Reply:
You're right, I'm not even sure how I ended up with that extra folder 🤦‍♂️

Reply:
Fixed in https://github.com/DataDog/sketches-dotnet/pull/2/commits/8110d28a5203a18810b0b0e47e83c9541617a4de